donkeycar/parts/info_overlay.py

```diff
@@ -0,0 +1,73 @@
+import cv2
+
+
+class InfoOverlayer(object):
+    """
+    Add a info overlay to the camera image
+    """
+
+    # Store the infos which will be write to image
+    info_list = []
```

Comment:
@DocGarbanzo It is used so that info can be written directly from other parts like this and have a shared `info_list`
```
from donkeycar.parts.info_overlay import InfoOverlayer
InfoOverlayer().add("hello")
```
"hello" can then be written onto the image along with other info like throttle or angle. However, it is supposed to be used as a temporary method in this case for easier debugging.